src/app.jsx

```diff
@@ -205,32 +291,16 @@ export default class App extends React.Component {
           <div className="inner two"></div>
           <div className="inner three"></div>
         </div>
+        <div className='toggle-user-data'>
+          <input type="radio" value="true" id="default"
+            onChange={this.toggleUserData.bind(this)}
```

Comment:
Change to ```this.toggleUserData``` because we bind this to toggleUserData in the constructor function ```    this.toggleUserData = this.toggleUserData.bind(this);```.